docs/Makefile

```diff
@@ -8,19 +8,24 @@ SPHINXBUILD   ?= uv run sphinx-build
 SOURCEDIR     = .
 BUILDDIR      = _build
 
+.PHONY: Makefile help clean rust-html uv-sync
+
```

Comment:
higher in the file but I think that the change to `uv run make` is the right thing to do, and `SPHINXBUILD` should just be `sphinx-build`

Reply:
ah okay, let me try that.

Reply:
Need this works.